### DISCUSSION TITLE
Can you voice conversations?

### DISCUSSION THREAD

**Discussion by Ishaname** [endless-sky/endless-sky#8219] [Q&A] [answered]
I was wondering if it is possible to voice the conversations you can get during missions. I have a friend of mine who was really wanting some voice work for a portfolio and also just thinking of adding it to my plugin or something. I don't think it's possible, but I figured I'd ask people smarter than me.

**Comment by Naf5000** [comment, marked as answer]
I don't think that feature is presently supported by the game, but if you wanted you could add it. The implementation would be a bit tricky, since narration means there can be pretty substantial gaps between spoken dialog, and I think it adds way too much work to adding new content to be a part of the main game, but as a plug-in it'd be fine and the capability for textboxes to play sounds could be used for other things.

On a different track, has your friend considered doing some recording for Librivox? Public domain audiobooks aren't video games, but it is still voicework.

**Reply by Ishaname** [reply]
Thanks for the input I'll have to find out how I might go about that now. I'll also let them know about Librivox, it sounds like a good start at least.
